### ISSUE TITLE
eslint-loader is not honoring `ignoreTemplateLiterals` in `max-len`

### ISSUE THREAD

**Issue by martynchamberlin** [webpack-contrib/eslint-loader#270]
We're using `eslint-loader` `2.1.2` and `eslint` `4.19.1`.

In our `.eslintrc.js` file we have this rule declared:

```js
    'max-len': ['error', {
      code: 120,
      ignoreUrls: true,
      ignoreStrings: true,
      ignoreComments: true,
      ignoreTemplateLiterals: true,
      ignoreRegExpLiterals: true,
    }],
```

In a `.vue` file we have this line, which is 123 lines long: 

```vue
      return `<iframe border="0" frameborder="0" allowtransparency="true" width="603" height="465" src="${src}"></iframe>`;
```

When we run `npm run lint`, this does not error, because of the above `ignoreTemplateLiterals: true` (here's [the documentation](https://eslint.org/docs/rules/max-len#ignoretemplateliterals) on the `ignoreTemplateLiterals`). If we were to remove that `ignoreTemplateLiterals` part, then `npm run lint` would fail (I've confirmed that this is indeed the case for us). 

However, when we run HMR using `eslint-loader`, we get this error for that `iframe` line above: 

>  http://eslint.org/docs/rules/max-len  Line 71 exceeds the maximum linelength of 120

To confirm that this line length limit constraint was indeed being specified from our `eslintrc.js` file and not from somewhere else, I temporarily changed `code: 120` to `code: 121` in `eslintrc.js`. When I cleared out  `node_modules/.cache/eslint-loader` and re-ran `npm run lint`, the error indeed changes: 

>  http://eslint.org/docs/rules/max-len  Line 71 exceeds the maximum linelength of 121

In other words, it seems that the `ignoreTemplateLiterals` is indeed not being honored by `eslint-loader`.
